### PR TITLE
Editorial: Use consistent wording for Records.

### PR DESCRIPTION
**Pull request by Ms2ger**

### CODE REVIEW

File: spec/duration.html [hunk 1]
```diff
@@ -900,7 +900,7 @@ <h1>BalanceDuration ( _days_, _hours_, _minutes_, _seconds_, _milliseconds_, _mi
           1. Set _nanoseconds_ to _nanoseconds_ modulo 1000.
         1. Else,
           1. Assert: _largestUnit_ is *"nanosecond"*.
-        1. Return the new Record {
+        1. Return the Record {
```
Comment (ljharb) [inline spec/duration.html:903]: the spec tends to omit the article for returns, a la https://tc39.es/ecma262/#sec-createlistiteratorRecord

Comment (Ms2ger) [reply spec/duration.html:903]: - "Return Record...": 1 hit
- "Return the Record...": 9 hits
- "Let x be Record...": no hits
- "Let x be the Record...": 13 hits

Comment (ljharb) [reply spec/duration.html:903]: obviously "be Record" would be wildly incorrect english; i guess i just was used to seeing that one hit :-) "Return the Record" seems like the better choice then.

**Review by ptomato — APPROVED** [review]
Seems like "Return the Record" is the most consistent.

**Pull request merged**